eal/cluster_configuration.ex

```diff
@@ -33,6 +33,15 @@ defmodule AntikytheraEal.ClusterConfiguration do
     This callback is called only at startup and the return value is passed to `RaftFleet.activate/1`.
     """
     @callback zone_of_this_host() :: String.t()
+
+    @doc """
+    Returns the number of seconds for which the initial health check to this host is delayed to
+    provide ample warm-up time for the host.
+
+    This callback is called only at startup and the return value is used to calculate the trial count for
+    establishing connections to other nodes.
+    """
+    @callback health_check_grace_period_in_seconds() :: non_neg_integer
```

Comment:
It looks like you should publish a new version after this review because you change this interface.
After that, you update the Antikythera version in mix.lock of antikythera_aws. It'll fix `@impl true` warning which you got.

Reply:
Thanks! It's necessary.
I will work according to this [ticket](https://acsmine.tok.access-company.com/redmine/issues/169054#note-12)

Reply:
@aYukiSekiguchi @aDaichiOta @Fumipo-Theta @ttokit 
I want to upgrade the a9a version in https://github.com/access-company/antikythera/pull/168/commits/5760d3218c4e8db356beeb4dbc734e1ba6d9e817
After this PR is merged to master, I will do the following:

- `git tag 0.4.0 <merge commit for this PR>`
- `git push origin 0.4.0`
- `mix hex.publish`
- Fix [this line](https://github.com/access-company/antikythera/blob/c7cd0a945e3b31f5e947571809e24bc057c5540f/mix.exs#L44) by `version: Antikythera.MixCommon.version_with_last_commit_info("0.4.1"),` to bump up.
- Fix [the a9a_aws PR](https://github.com/access-company/antikythera_aws/pull/8) using a9a v0.4.0 and merge it to master
- Update a9a_acs dependency on a9a and a9a_aws.